packages/server/src/channels/api.ts

```diff
@@ -5,8 +5,39 @@ export class ChannelApi {
   constructor(private router: Router, private app: App) {}
 
   async setup() {
+    const whRouter = Router()
```

Comment:
wh? What does it mean?

Reply:
wh = webhook

I changed it to webhookRouter